File: all_sum.py
import pandas as pd

# Membuat dataframe
df = pd.DataFrame({
    'A': [1, 2, 3],
    'B': [4, 5, 6]
})

# Menampilkan dataframe
print(df)
print(df.values.sum())
print()

print(df.sum())
print(df.sum(axis=1))
print(df['A'].sum())
print(df.loc[:, 'A'].sum())
print(df.loc[0, :].sum())

File: all_sum.r
# Membuat dataframe contoh
df <- data.frame(
    A = c(1, 2, 3),
    B = c(4, 5, 6)
)

# Menampilkan dataframe
print(df)
cat('\n\n')
print(colSums(df))
print(rowSums(df))

cat('\n\n')
print(sum(df))
print(sum(df$A))
print(sum(df[, 'A']))
print(sum(df[1,]))